### ISSUE TITLE
The Circuit is not accounte for Forward Euler

### ISSUE THREAD

**Issue by MartinGuillonLPP** [federicopetronio/FLHET1D#3]
In the script FLHET_compiled.py, the block
if TIMESCHEME == "Forward Euler"  :
    ...

doesn't contain any sub-block 
    if Circuit:
         ....
that recomputes the voltage V.

It is done in the block RK3 though.
So at the end, the two methods of time integration would not give the same results if the Circuit is on.

**Comment by federicopetronio** [comment]
True. Adding the circuit in the forward Euleur seems a straight-forward task, though

**Comment by MartinGuillonLPP** [comment]
> True. Adding the circuit in the forward Euleur seems a straight-forward task, though

Yes right. Actually there is this problem in all the branches. I will correct it in the "implicit_heat_flux" at least. I encountered it by looking the cause of differences between matthieu results and mine.
